sway-core/src/semantic_analysis/namespace/namespace.rs

```diff
@@ -290,6 +290,43 @@ impl Namespace {
             parent_mod_path,
         }
     }
+
+    #[allow(clippy::too_many_arguments)]
+    pub(crate) fn insert_trait_implementation<'a>(
+        &mut self,
+        trait_name: CallPath,
+        trait_type_args: Vec<TypeArgument>,
+        type_id: TypeId,
+        methods: &[DeclarationId],
+        impl_span: &Span,
+        is_impl_self: bool,
+        engines: Engines<'a>,
+    ) -> CompileResult<()> {
+        let full_trait_name = trait_name.to_fullpath(self);
+
+        self.implemented_traits.insert(
+            full_trait_name,
+            trait_type_args,
+            type_id,
+            methods,
+            impl_span,
+            is_impl_self,
+            engines,
+        )
+    }
+
+    pub(crate) fn get_methods_for_type_and_trait_name(
+        &mut self,
+        engines: Engines<'_>,
+        type_id: TypeId,
+        trait_name: &CallPath,
+    ) -> Vec<DeclarationId> {
+        // Use trait name with module path as this is expected in get_methods_for_type_and_trait_name
```

Comment:
Can you add this same comment to the `insert_trait_implementation` method above?

Reply:
Added comment to both methods.